src/components/RequestsFilters/RequestsFilters.test.js

```diff
@@ -255,4 +255,66 @@ describe('RequestsFilters', () => {
       }), {});
     });
   });
+
+  describe('When activeFilters prop is empty', () => {
+    const propsWithoutFilters = {
+      ...props,
+      activeFilters: {},
+      titleLevelRequestsFeatureEnabled: true,
+    };
+    const checkboxFilterCallOrder = {
+      requestTypeCheckbox: 1,
+      requestStatusCheckbox: 2,
+    };
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+
+      render(
+        <RequestsFilters
+          {...propsWithoutFilters}
+        />
+      );
+    });
+
+    it('should trigger MultiSelectionFilter with correct selectedValues prop', () => {
```

Comment:
Could you please clarify whether it would be better to rename to "should trigger TagsFilter with correct selectedValues prop"?